src/apps/mailing/services.py

```diff
@@ -62,8 +62,11 @@ async def send(self, message: MessageSchema) -> None:
         fm = mailing_class(self._connection)
         await fm.send_message(message)
 
-    def get_template(self, path: str, **kwargs):
-        template = self.env.get_template(f"{path}.html")
-        html = template.render(**kwargs)
-
-        return html
+    def get_localized_html_template(self, _template_name: str, _language: str, **kwargs) -> str:
+        kwargs["language"] = _language
+        try:
```

Comment:
why use this _language parameter and _template_name? 
for language we can get it from kwargs list, kwargs.get('language')
other solution, it to receive this as "language" and pass it to render() function like this render(language=language, **kwargs)

in python leading underscore denote a private variable, we don't use it in functions arguments

Reply:
I can remove the underscore from the parameter name.

But I also updated this function with an extra feature, which is if the request (template name + language) template file is unavailable, this function will return the English template file. This is why I needed "language" as a separate parameter for this function, so it can recursively call itself with `language="en"` in the fallback scenario.

I also already updated all the places that this function is called, so `kwargs` no longer contains a `language` field. Instead, the `language` field is now injected into `kwargs` from the `language` parameter. I feel having both "template name" and "language" as distinctly required parameter for this function is more clear, because now everything in `kwargs` is template-dependent, whereas "template name" and "language" are important for fetching the template file.